frontend/pages/schedule/ManageSchedulePage/components/PreviewDataModal/PreviewDataModal.tsx

```diff
@@ -0,0 +1,87 @@
+/* This component is used for creating and editing both global and team scheduled queries */
+
+import React from "react";
+import { syntaxHighlight } from "fleet/helpers";
+
+import ReactTooltip from "react-tooltip";
+import Modal from "components/Modal";
+import Button from "components/buttons/Button";
+
+import QuestionIcon from "../../../../../../assets/images/icon-question-16x16@2x.png";
+
+const baseClass = "preview-data-modal";
+
+interface IPreviewDataModalProps {
+  onCancel: () => void;
+}
+
+const PreviewDataModal = ({
+  onCancel,
+}: IPreviewDataModalProps): JSX.Element => {
+  const json = {
+    action: "snapshot",
+    snapshot: [
+      {
+        remote_address: "0.0.0.0",
+        remote_port: "0",
+        cmdline: "/usr/sbin/syslogd",
+      },
+    ],
+    name: "xxxxxxx",
+    hostIdentifier: "xxxxxxx",
+    calendarTime: "xxx xxx  x xx:xx:xx xxxx UTC",
+    unixTime: "xxxxxxxxx",
+    epoch: "xxxxxxxxx",
+    counter: "x",
+    numerics: "x",
+  };
```

Comment:
I'm thinking this might be why you were getting type errors on the `JSON` type. In order for this to be valid JSON, the object properties all need to be wrapped in quotes.

Reply:
The syntax highlighter auto does it for us, if you try to save with the quotes in typescript, it will auto remove them. :)